next.config.js

```diff
@@ -1,35 +1,52 @@
-const withCSS = require("@zeit/next-css");
+// @ts-check
+
 const MonacoWebpackPlugin = require("monaco-editor-webpack-plugin");
-const withFonts = require("next-fonts");
+const assert = require("assert");
```

Comment:
would like to use `invariant` instead but `assert` works just fine, just without arg interpolation